src/c++/perf_analyzer/genai-perf/genai_perf/llm_metrics.py

```diff
@@ -73,8 +82,8 @@ class Metrics:
 
     def __init__(
         self,
-        request_throughputs: list[float] = [],
-        request_latencies: list[int] = [],
+        request_throughputs: typing.List[float] = [],
```

Comment:
Any reason not to just import `List`? I think it would be less verbose and more clear.
```suggestion
        request_throughputs: List[float] = [],
```

Reply:
Thanks for this suggestion! Fixed it. I was focused on fixing the type warnings and did not think to do it this way here. It's more consistent with our other files too.